<!doctype html>
<head>
    <title>ARTFL Browser</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!--
    <link rel="stylesheet" type="text/css" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css" />
    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
    -->
   

    <link rel=stylesheet type=text/css href="/static/typeaheadjs.css">
    <link rel=stylesheet type=text/css href="/static/css/font-awesome.min.css">
    <link href='https://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
    <script type="text/javascript" src="/static/jquery-1.11.1.min.js"></script>
    <script type="text/javascript" src="/static/bootstrap.min.js"></script>
    <script type="text/javascript" src="/static/typeahead.bundle.min.js"></script>

    <script type="text/javascript" src="/static/jquery.tablesorter.min.js"></script>
    <script type="text/javascript" src="/static/jquery.tablesorter.pager.js"></script>
    <link rel=stylesheet type=text/css href="/static/theme.default.css">
    <script type="text/javascript" src="/static/underscore-min.js"></script>

    <link rel=stylesheet type=text/css href="/static/bootstrap.min.css">
    <link rel=stylesheet type=text/css href="/static/style.css">
    <script type="text/javascript" src="/static/d3.v3.min.js"></script>
    <script type="text/javascript" src="https://gabelerner.github.io/canvg/rgbcolor.js"></script>
    <script type="text/javascript" src="https://gabelerner.github.io/canvg/StackBlur.js"></script>
    <script type="text/javascript" src="https://gabelerner.github.io/canvg/canvg.js"></script>

    <script type="text/javascript" src="/static/index.js"></script>
    <script type="text/javascript" src="/static/exac.js"></script>

    <script src="https://code.highcharts.com/highcharts.js"></script>
    <script src="https://code.highcharts.com/modules/exporting.js"></script>



    <!--
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-55729303-1', 'auto');
        ga('send', 'pageview');
    </script>-->

    <script type="text/javascript">
        number_of_samples = 60706;
        release_number = 0.3;
        number_of_samples_full = 91796;
        $(document).ready(function() {
            $('.number_samples').html(Number(number_of_samples).toLocaleString('en'));
            $('.number_samples_full').html(Number(number_of_samples_full).toLocaleString('en'));
            $('.release_number').html(Number(release_number).toLocaleString('en'));
        });
        $(function() {
            var bestPictures = new Bloodhound({
              datumTokenizer: Bloodhound.tokenizers.obj.whitespace('value'),
              queryTokenizer: Bloodhound.tokenizers.whitespace,
              remote: '/autocomplete/%QUERY'
            });

            bestPictures.initialize();

            $('.awesomebar').typeahead(
                {
                    autoselect: true,
                },
                {
                    name: 'best-pictures',
                    displayKey: 'value',
                    source: bestPictures.ttAdapter(),
                }
            );
            $('.awesomebar').bind('typeahead:selected', function(obj, datum) {
                window.location.href = '/awesome?query=' + datum.value;
            });
        });

    </script>
</head>
<body>
<nav class="navbar navbar-default " role="navigation" style="background: white; ">
    <div class="container-fluid row " style="margin-right: 0px;">
        <div class="navbar-header col-md-3 col-xs-3">
            <button type="button" class="navbar-toggle pull-right" data-toggle="collapse" data-target="#navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/" ><img src="/static/images/Mergelogo.png" align="left" height="50" width="100"/> <span style="position:absolute; top:40px; left: 100px; color: black; font-weight: bold; float: left; vertical-align:bottom; font-size: 15px; font-family:'Lato' !important;">ARTFL Sequence Analysis </span></a>

        </div>
        <div class="col-md-5 col-xs-5"  style="margin-top: 30px; ">
            <div id="navbar_form_container">
                <form action="/awesome" class="navbar-form" role="search">
                    <div class="form-group" id="navbar-awesomebar">
                        <input type="submit" style="display: none;"/>
                        <input id="navbar-searchbox-input" name="query" class="form-control awesomebar" type="text" placeholder="Gene, transcript, variant, or region"/>
                        <input type="submit" style="position: absolute; left: -9999px"/>
                    </div>
                </form>
            </div>
        </div>
        <div class="col-md-4 col-xs-4" style="margin-top: 30px;"> 
            <div class="nav navbar-nav navbar-right text-right" >
                    <!--<p class="text-muted" style="font-family:'Lato' !important; position:absolute; text-align: right; top:50px; left:85%;">Original source code from <a href="http://exac.broadinstitute.org" target="_blank">ExAC</a> </p>-->
                    <ul class="list-unstyled" style=" font-family:'Lato' !important; ">
                        <li style="font-size: 12px;">Original source code from <a href="http://exac.broadinstitute.org" target="_blank">ExAC</a></li>
                        <li style="font-size: 12px;">Implemented by <span style="color:#245580;" >Sandeep Deverasetty, </span> <a href="https://www.semel.ucla.edu/coppolalab" target="_blank"> Coppola lab </a> </li>
                    </ul>
            </div>
        </div>

    </div>
</nav>







    <!-- Render context vars in JS here -->
    <script type="text/javascript">
        window.gene = {"canonical_transcript": "ENST00000406386", "chrom": "22", "full_gene_name": "TRIO and F-actin binding protein", "gene_id": "ENSG00000100106", "gene_name": "TRIOBP", "gene_name_upper": "TRIOBP", "omim_accession": "609761", "omim_description": " TRIO- AND F-ACTIN-BINDING PROTEIN; TRIOBP", "other_names": ["DFNB28", "HRIHFB2122", "KIAA1662", "TARA"], "start": 38093012, "stop": 38172564, "strand": "+", "xstart": 22038093012, "xstop": 22038172564};
        window.transcript = {"chrom": "22", "exons": [{"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38093012, "stop": 38093097, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093012, "xstop": 22038093097}, {"chrom": "22", "feature_type": "UTR", "gene_id": "ENSG00000100106", "start": 38093012, "stop": 38093097, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093012, "xstop": 22038093097}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38093596, "stop": 38093704, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093596, "xstop": 22038093704}, {"chrom": "22", "feature_type": "UTR", "gene_id": "ENSG00000100106", "start": 38093596, "stop": 38093704, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093596, "xstop": 22038093704}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38097314, "stop": 38097487, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038097314, "xstop": 22038097487}, {"chrom": "22", "feature_type": "UTR", "gene_id": "ENSG00000100106", "start": 38097314, "stop": 38097373, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038097314, "xstop": 22038097373}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38097374, "stop": 38097487, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038097374, "xstop": 22038097487}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38106435, "stop": 38106574, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038106435, "xstop": 22038106574}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38106435, "stop": 38106574, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038106435, "xstop": 22038106574}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38109218, "stop": 38109419, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038109218, "xstop": 22038109419}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38109218, "stop": 38109419, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038109218, "xstop": 22038109419}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38111771, "stop": 38111942, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038111771, "xstop": 22038111942}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38111771, "stop": 38111942, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038111771, "xstop": 22038111942}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38119193, "stop": 38122511, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038119193, "xstop": 22038122511}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38119193, "stop": 38122511, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038119193, "xstop": 22038122511}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38129306, "stop": 38129420, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038129306, "xstop": 22038129420}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38129306, "stop": 38129420, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038129306, "xstop": 22038129420}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38130407, "stop": 38131450, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038130407, "xstop": 22038131450}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38130407, "stop": 38131450, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038130407, "xstop": 22038131450}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38134650, "stop": 38134727, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038134650, "xstop": 22038134727}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38134650, "stop": 38134727, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038134650, "xstop": 22038134727}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38136903, "stop": 38137040, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038136903, "xstop": 22038137040}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38136903, "stop": 38137040, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038136903, "xstop": 22038137040}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38147780, "stop": 38147836, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038147780, "xstop": 22038147836}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38147780, "stop": 38147836, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038147780, "xstop": 22038147836}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38150885, "stop": 38150992, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038150885, "xstop": 22038150992}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38150885, "stop": 38150992, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038150885, "xstop": 22038150992}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38151109, "stop": 38151198, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038151109, "xstop": 22038151198}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38151109, "stop": 38151198, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038151109, "xstop": 22038151198}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38151558, "stop": 38151667, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038151558, "xstop": 22038151667}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38151558, "stop": 38151667, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038151558, "xstop": 22038151667}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38153621, "stop": 38154146, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038153621, "xstop": 22038154146}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38153621, "stop": 38154146, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038153621, "xstop": 22038154146}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38155162, "stop": 38155272, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038155162, "xstop": 22038155272}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38155162, "stop": 38155272, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038155162, "xstop": 22038155272}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38161678, "stop": 38161825, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038161678, "xstop": 22038161825}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38161678, "stop": 38161825, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038161678, "xstop": 22038161825}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38164082, "stop": 38164184, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038164082, "xstop": 22038164184}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38164082, "stop": 38164184, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038164082, "xstop": 22038164184}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38165036, "stop": 38165195, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038165036, "xstop": 22038165195}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38165036, "stop": 38165195, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038165036, "xstop": 22038165195}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38165270, "stop": 38165383, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038165270, "xstop": 22038165383}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38165270, "stop": 38165383, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038165270, "xstop": 22038165383}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38167658, "stop": 38167744, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038167658, "xstop": 22038167744}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38167658, "stop": 38167744, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038167658, "xstop": 22038167744}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38168609, "stop": 38168772, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038168609, "xstop": 22038168772}, {"chrom": "22", "feature_type": "CDS", "gene_id": "ENSG00000100106", "start": 38168609, "stop": 38168767, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038168609, "xstop": 22038168767}, {"chrom": "22", "feature_type": "UTR", "gene_id": "ENSG00000100106", "start": 38168768, "stop": 38168772, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038168768, "xstop": 22038168772}, {"chrom": "22", "feature_type": "exon", "gene_id": "ENSG00000100106", "start": 38169791, "stop": 38172564, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038169791, "xstop": 22038172564}, {"chrom": "22", "feature_type": "UTR", "gene_id": "ENSG00000100106", "start": 38169791, "stop": 38172564, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038169791, "xstop": 22038172564}], "gene_id": "ENSG00000100106", "start": 38093012, "stop": 38172564, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093012, "xstop": 22038172564};
        window.table_variants = [];
        window.variants_in_transcript = [];
        window.transcripts_in_gene = [{"chrom": "22", "gene_id": "ENSG00000100106", "start": 38093012, "stop": 38172564, "strand": "+", "transcript_id": "ENST00000406386", "xstart": 22038093012, "xstop": 22038172564}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38093629, "stop": 38122578, "strand": "+", "transcript_id": "ENST00000492485", "xstart": 22038093629, "xstop": 22038122578}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38093629, "stop": 38170056, "strand": "+", "transcript_id": "ENST00000344404", "xstart": 22038093629, "xstop": 22038170056}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38142236, "stop": 38156354, "strand": "+", "transcript_id": "ENST00000407319", "xstart": 22038142236, "xstop": 22038156354}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38142241, "stop": 38170148, "strand": "+", "transcript_id": "ENST00000403663", "xstart": 22038142241, "xstop": 22038170148}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38142278, "stop": 38154143, "strand": "+", "transcript_id": "ENST00000428075", "xstart": 22038142278, "xstop": 22038154143}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38142330, "stop": 38153971, "strand": "+", "transcript_id": "ENST00000413051", "xstart": 22038142330, "xstop": 22038153971}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38147005, "stop": 38153990, "strand": "+", "transcript_id": "ENST00000418339", "xstart": 22038147005, "xstop": 22038153990}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38147393, "stop": 38151582, "strand": "+", "transcript_id": "ENST00000452519", "xstart": 22038147393, "xstop": 22038151582}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38147406, "stop": 38154055, "strand": "+", "transcript_id": "ENST00000417857", "xstart": 22038147406, "xstop": 22038154055}, {"chrom": "22", "gene_id": "ENSG00000100106", "start": 38150890, "stop": 38151518, "strand": "+", "transcript_id": "ENST00000331103", "xstart": 22038150890, "xstop": 22038151518}];
        window.coverage_stats = [];
        window.page_name = window.gene.gene_id;
    </script>
    <style>

    </style>
    <script>
        $(document).ready(function() {
            $("#variants_loading").hide();
            $("#variants_table_container").show();
        });
    </script>
    <div class="container-fluid">
        <div class="col-md-10 col-xs-offset-1 col-md-offset-1">
            <h1>Gene: TRIOBP</h1>
            <hr/>
        </div>
        <div class="row">
            <div class="col-md-4 col-xs-10 col-xs-offset-1 col-md-offset-0">
                <dl class="dl-horizontal">
                    
                        <dt>TRIOBP</dt>
                        <dd>TRIO and F-actin binding protein</dd>
                    
                    <dt>Number of variants</dt>
                    <dd><span id="number_of_variants"></span> (Including filtered: 0)</dd>


                    <script>
                        filtered_variants = _.filter(window.table_variants, function(d) {
                            return d.filter == 'PASS';
                        });
                        $('#number_of_variants').replaceWith(filtered_variants.length);
                        lof_filtered_variants = _.filter(window.table_variants, function(d) {
                            return d.filter == 'PASS' && d.category == 'lof_variant';
                        });
                        $('#number_of_lof_variants').replaceWith(lof_filtered_variants.length);
                    </script>


                    <dt class="hidden-xs">UCSC Browser</dt>
                    <dd class="hidden-xs">
                        <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr22%3A38093011-38172563" target="_blank">
                            22:38093011-38172563
                            <i class="fa fa-external-link"></i>
                        </a>
                    </dd>
                    <dt class="hidden-xs">GeneCards</dt>
                    <dd class="hidden-xs">
                        <a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=TRIOBP" target="_blank">
                            TRIOBP
                            <i class="fa fa-external-link"></i>
                        </a>
                    </dd>
                    
                        <dt>OMIM</dt>
                        <dd>
                            <a href="http://omim.org/entry/609761" target="_blank">
                                TRIOBP
                                <i class="fa fa-external-link"></i>
                            </a>
                        </dd>
                    
                    <dt>Other</dt>
                    <dd>
                        <div class="dropdown">
                            <button class="btn btn-default dropdown-toggle" type="button" id="external_ref_dropdown" data-toggle="dropdown">
                                External References
                                <span class="caret"></span>
                            </button>
                            <ul class="dropdown-menu" role="menu" aria-labelledby="external_ref_dropdown">
                                <li role="presentation" class="visible-xs-block">
                                    <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr22%3A38093011-38172563" target="_blank">
                                        UCSC Browser
                                        <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation" class="visible-xs-block">
                                    <a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=TRIOBP" target="_blank">
                                        GeneCards
                                        <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation">
                                    <a role="menuitem" tabindex="-1" href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100106" target="_blank">
                                        Ensembl <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation">
                                    <a role="menuitem" tabindex="-1" href="http://en.wikipedia.org/TRIOBP" target="_blank">
                                        Wikipedia <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation">
                                    <a role="menuitem" tabindex="-1" href="http://www.ncbi.nlm.nih.gov/pubmed?term=TRIOBP" target="_blank">
                                        PubMed Search <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation">
                                    <a role="menuitem" tabindex="-1" href="http://www.wikigenes.org/?search=TRIOBP" target="_blank">
                                        Wikigenes <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                                <li role="presentation">
                                    <a role="menuitem" tabindex="-1" href="http://www.gtexportal.org/home/gene/TRIOBP" target="_blank">
                                        GTEx (Expression) <i class="fa fa-external-link"></i>
                                    </a>
                                </li>
                            </ul>
                        </div>
                    </dd>
                </dl>
            </div>
            <div class="col-md-1 hidden-xs">
                <div class="dropdown">
                    <button class="btn btn-default dropdown-toggle" type="button" id="transcript_dropdown" data-toggle="dropdown">
                        Transcripts
                        <span class="caret"></span>
                    </button>
                    <ul class="dropdown-menu" role="menu" aria-labelledby="transcript_dropdown">
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000406386" target="_blank">
                                    ENST00000406386
                                    
                                        *
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000492485" target="_blank">
                                    ENST00000492485
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000344404" target="_blank">
                                    ENST00000344404
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000407319" target="_blank">
                                    ENST00000407319
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000403663" target="_blank">
                                    ENST00000403663
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000428075" target="_blank">
                                    ENST00000428075
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000413051" target="_blank">
                                    ENST00000413051
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000418339" target="_blank">
                                    ENST00000418339
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000452519" target="_blank">
                                    ENST00000452519
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000417857" target="_blank">
                                    ENST00000417857
                                    
                                </a>
                            </li>
                        
                            <li role="presentation">
                                <a role="menuitem" tabindex="-1" href="/transcript/ENST00000331103" target="_blank">
                                    ENST00000331103
                                    
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <div class="col-md-5 col-md-offset-1">
                
                    <table class="table">
                        <thead>
                            <tr>
                                <th style="padding-top: 4px;">Constraint <br/>from ExAC</th>
                                <th style="padding-top: 4px; width=50%;">Expected <br/>no. variants</th>
                                <th style="padding-top: 4px;" class="tooltip-table-header" data-tooltip="Note that these numbers only consider &#xa; rare (AF < 0.001) variation &#xa; Not all variants in the table &#xa; below may be included">Observed <br/>no. variants in ExAC</th>
                                <th style="padding-top: 4px;" class="tooltip-table-header" data-tooltip="Constraint Metric for each &#xa; category (see FAQ) &#xa; synonymous: Z score &#xa; missense: Z score&#xa; LoF: pLI">Constraint<br/>Metric</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>Synonymous</td>
                                <td>354.1</td>
                                <td>379</td>
                                <td>z = 
                                        -0.82
                                    
                                </td>
                            </tr>
                            <tr>
                                <td>Missense</td>
                                <td>864.8</td>
                                <td>918</td>
                                <td>z = 
                                        -0.89
                                    
                                </td>
                            </tr>
                            <tr>
                                <td class="tooltip-table-header" data-tooltip="Stop-gained and essential splice site">LoF</td>
                                <td>81.4</td>
                                <td>35</td>
                                <td class="tooltip-table-header" data-tooltip="Probability of LoF intolerance (see FAQ)">
                                        pLI
                                     =
                                    0.00
                                </td>
                            </tr>
                        </tbody>
                    </table>
                
            </div>
        </div>
        <div class="row">
            <div class="col-md-10 col-md-offset-1">
                <h3>Gene summary</h3>
                <h4>(<span class="hidden-xs">Coverage shown for </span><a href="http://www.ensembl.org/Help/Glossary?id=346" target="_blank">canonical transcript</a>: ENST00000406386)</h4>
                <div class="row">
                    
                        <dl class="dl-horizontal">
    <dt><span style="text-transform:capitalize" id="avg_coverage_type">Mean</span> coverage</dt>
    <dd><span id="avg_coverage"></span></dd>
</dl>
<div id="gene_plot">
    <span class="hidden-xs">
        <label for="display_coverage_metric_group">
            Display:
        </label>
        <span class="btn-group" data-toggle="buttons" id="display_coverage_metric_group">
            <button class="btn btn-primary active display_coverage_metric_buttons"
                    id="display_coverage_overview_button" data-tooltip="Shows all exons">
                <input type="radio">Overview</input>
            </button>
            <button class="btn btn-primary display_coverage_metric_buttons"
                    id="display_coverage_detail_button" data-tooltip="Zoomed in view (can scroll horizontally if needed)">
                <input type="radio">Detail</input>
            </button>
        </span>
        <label>
            <input type="checkbox" id="include_utrs_checkbox" value=""> Include UTRs in plot
        </label>
    </span>
    <span style="float:right;" class="hidden-xs">
    <span style="float: right;">
        <label for="coverage_metric_group">
            Coverage metric:
        </label>
        <span class="btn-group" data-toggle="buttons" id="coverage_metric_group">
            <button class="btn btn-primary active coverage_metric_buttons"
                    id="covered_mean_covmet_button" data-tooltip="Per-base depth of &#xa;coverage. Capped at 100X">
                    <input type="radio">Average</input>
            </button>
            <button class="btn btn-primary coverage_metric_buttons" id="covered_covmet_button"
                data-tooltip="Fraction of individuals with&#xa;coverage above X at each base.&#xa;Use drop down below to select X.">
                <input type="radio">Individuals over X</input>
            </button>
        </span>
    </span>
    <br/>
    <div class="coverage_subcat_selectors" id="over_x_select_container" style="display: none; float:right;">
        <label for="over_x_select">
            Individuals over:
        </label>
        <select id="over_x_select">
            <option>1X</option>
            <option>5X</option>
            <option>10X</option>
            <option>15X</option>
            <option>20X</option>
            <option>25X</option>
            <option selected>30X</option>
            <option>50X</option>
            <option>100X</option>
        </select>
    </div>
    <div class="coverage_subcat_selectors" id="average_select_container" style="float:right;">
        <label for="average_select">
            Metric:
        </label>
        <select id="average_select">
            <option selected>mean</option>
            <option>median</option>
        </select>
    </div>
</span>
    <br/>
    <div id="gene_plot_container_container">
        <div id="loading_coverage">
            Loading coverage...
            <br/><br/>
        </div>
        <span id="gene_plot_axis_container" class="hidden-xs" style="width:50px; display: inline-block;"></span>
        <span id="gene_plot_container" style="overflow-x:scroll; display: inline-block;"></span>
        <a class="btn btn-success" id="coverage_plot_download" target="_blank" download="TRIOBP_coverage">Save coverage plot</a>
        <a class="btn btn-success" id="exon_plot_download" target="_blank" download="TRIOBP_exon">Save exon image</a>
    </div>
</div>
<div id="not_covered" style="display: none;">
    No coverage for this transcript.
</div>
<br/>
<script>
function gene_chart(data, new_data, variant_data, _transcript) {
    var coords = 'pos_coding_noutr';
    var coding_coordinate_params = get_coding_coordinate_params(_transcript, true);
    var chart_width = gene_chart_width;
    var metric = 'mean';

    var exon_x_scale = d3.scale.linear()
        .domain([0, coding_coordinate_params.size])
        .range([0, chart_width]);

    var max_cov = (metric == 'mean' || metric == 'median') ? 300 : 1; //sandeep
    var y = d3.scale.linear()
        .domain([0, max_cov])
        .range([gene_chart_height, 0]);
    
    var yAxis = d3.svg.axis()
        .scale(y)
        .orient("left");
    console.log(yAxis)
    var svg = d3.select('#gene_plot_container').append("svg")
        .attr("width", chart_width + gene_chart_margin.left + gene_chart_margin.right)
        .attr("height", gene_chart_height + gene_chart_margin.top + gene_chart_margin.bottom)
        .attr('id', 'inner_svg')
        .attr('class', 'hidden-xs')
        .append("g")
        .attr('id', 'inner_graph')
        .attr("transform", "translate(" + gene_chart_margin.left + "," + gene_chart_margin.top + ")");

    var area = d3.svg.area()
        .x( function(d) {
            return exon_x_scale(d[coords]);
        }).y0( function(d) {
            return gene_chart_height;
        }).y1( function(d) {
            return (metric in d) ? y(d[metric]) : gene_chart_height;
        });

    svg.append("path")
        .datum(new_data)
        .style("fill", "steelblue")
        .attr('class', 'area')
        .attr("d", area);

    svg.append("g")
        .attr("class", "y axis")
        .call(yAxis);

    svg.selectAll("circle")
        .data(data)
        .enter()
        .append("circle")
        .attr("cx", function(d) {
            if (d['pos_coding_noutr'] == undefined) {
                return -100;
            } else {
                return exon_x_scale(d[coords]);
            }
        })
        .attr("cy", gene_chart_height + 3)
        .attr('r', 2 );

    d3.select('#gene_plot_container').append("br"); //make sure track_svg is below graph and not to the right of it

    // plot exons
    var svg_outer = d3.select('#gene_plot_container').append("svg")
        .attr("width", chart_width + gene_chart_margin_lower.left + gene_chart_margin_lower.right)
        .attr("height", lower_gene_chart_height)
        .attr('id', 'track_svg')
        .append("g")
        .attr('id', 'track')
        .attr("transform", "translate(" + gene_chart_margin_lower.left + "," + 0 + ")");

    var exon_color = "lightsteelblue";
    svg_outer.append("line")
        .attr("y1", lower_gene_chart_height/2)
        .attr("y2", lower_gene_chart_height/2)
        .attr("x1", 0)
        .attr("x2", exon_x_scale(coding_coordinate_params.size))
        .attr('id', 'boundary_line')
        .attr("stroke-width", 5)
        .attr("stroke", exon_color);

    console.log(_transcript);
    // plot exon rects
    svg_outer.selectAll("bar")
        .data(_transcript.exons)
        .enter()
        .append("rect")
        .attr('class', 'track_bar')
        .style("fill", exon_color)
        .attr("x", function(d, i) { return exon_x_scale(get_coding_coordinate(_transcript, d.start, true)); })
        .attr("y", function(d, i) {
            if (d.feature_type == 'CDS') {
                return 0;
            } else {
                return lower_gene_chart_height/4;
            }
        })


        .attr("width", function(d, i) {
            if (get_coding_coordinate(_transcript, d.start, true) == -100) {
                return exon_x_scale(175);
            }
            return exon_x_scale(d.stop-d.start+1);
        })
        .attr("height", function(d, i) {
            if (d.feature_type == 'CDS') {
                return lower_gene_chart_height;
            } else {
                return lower_gene_chart_height/2;
            }
        });


    var a_s = _transcript.strand == "-"? -1 : 1; //arrow direction
    var a_x = -5;  //arrow position on x-axis
    var a_y = lower_gene_chart_height/2.0; //arrow position on y-axis
    var a_w = 2; //arrow width
    var points = [[a_x+a_s*6, a_y], [a_x+a_s*1, a_y+a_w*3], [a_x+a_s*1, a_y+a_w], [a_x-a_s*9, a_y+a_w],
        [a_x-a_s*9, a_y-a_w], [a_x+a_s*1, a_y-a_w], [a_x+a_s*1, a_y-a_w*3]];
    svg_outer.append("polygon")
            .attr("points", points.join(" "))
            .attr("fill", "steelblue")
            .attr("stroke", "black");

    var bounds = get_af_bounds(variant_data);
    var min_af = bounds[0];
    var max_af = bounds[1];
    var variant_size_scale = d3.scale.log()
        .domain([min_af, max_af])
        //Circle/Ellipse
        .range([2, lower_gene_chart_height/3]);
        //Rectangle
//        .range([lower_gene_chart_height, 2]);

    // show variant category on hover
    var tip = d3.tip().attr('class', 'd3-tip').html(function(d) {
        if (d.category) {
            var csq = d.major_consequence.replace('_variant', '')
                    .replace('_', ' ')
                    .replace('utr', 'UTR')
                    .replace('3 prime', "3'")
                    .replace('5 prime', "5'")
                    .replace('nc ', "non-coding ");
            var output = csq + '<br/>' + d.chrom + ':' + d.pos + ' ' + d.ref + '&#8594;' + d.alt;
            if (d.major_consequence == 'missense_variant' || d.major_consequence == 'synonymous_variant') {
                output += '<br/>' + d.HGVSp;
            }
            output += '<br/>Frequency: ' + d.allele_freq.toPrecision(3);
            return output;
        } else {
            return 'None';
        }
    });
    svg.call(tip);

    // plot variants
    svg_outer.selectAll("bar")
        .data(variant_data)
        .enter()
        .append("a")
        .attr('class', 'track_variant_link')
        .attr("xlink:href", function(d, i) { return "/variant/" + d.chrom + "-" + d.pos + "-" + d.ref + "-" + d.alt; })
        .attr("data-toggle", "tooltip")
        .attr('filter_status', function(d) {
            return d.filter;
        })
        .attr('category', function(d) {
            return d.category;
        })
        .attr('variant_id', function(d) {
            return d.variant_id;
        })
        .on('mouseover', function(d) {
            $('#variant_' + d.variant_id).find('td').addClass('table_hover');
            tip.show(d);
        })
        .on('mouseout', function(d) {
            $('#variant_' + d.variant_id).find('td').removeClass('table_hover');
            tip.hide(d);
        })
        //Circle
//        .append("circle")
        //Ellipse
        .append("ellipse")
        .attr("class", function(d) {
            return "track_variant " + d.category;
        })
        .style("opacity", 0.5)
        .attr("cx",  function(d) {
            if (d['pos_coding_noutr'] == undefined) {
                return -100;
            } else {
                return exon_x_scale(d[coords]);
            }
        })
        .attr("cy", lower_gene_chart_height/2)
        //Circle
//        .attr("r", function(d, i) { return variant_size_scale(d.allele_freq); })
        //Ellipse
        .attr("rx", 2)
        .attr("ry", function(d, i) {
            if (!d.allele_freq) {
                return 0;
            } else {
                return variant_size_scale(d.allele_freq);
            }
        })
        // Workaround for exporting d3 to SVG (other delcaration in style.css).
        .attr('stroke', variant_colors)
        .attr('fill', variant_colors);
        //Rectangle
//        .append("rect")
//        .attr("class", "track_variant")
//        .style("fill", "darkred")
//        .style("opacity", 0.5)
//        .attr("x", function(d, i) {
//            var tx_coord = d.transcript_coordinates[transcript];
//            if (tx_coord == 0) {
//                return -1000;
//            } else {
//                var variant_exon_number = d.vep_annotations[0]['EXON'].split('/')[0] - 1;
//                return exon_x_scale(tx_coord + variant_exon_number*padding);
//            }
//        })
//        .attr("y", function(d, i) { return lower_gene_chart_height/2 - variant_size_scale(d.allele_freq)/2; } )
//        .attr("width", 2)
//        .attr("height", function(d, i) { return variant_size_scale(d.allele_freq); })
//        .attr("rx", 6)
//        .attr("ry", 6);

}

function variant_colors(d) {
    if (d.category == 'lof_variant') {
        return '#cd2932';
    } else if (d.category == 'missense_variant') {
        return '#a96500';
    } else if (d.category == 'synonymous_variant') {
        return '#157e28';
    }
}

function change_coverage_chart(data, new_data, variant_data, _transcript, scale_type, metric, skip_utrs, container) {
    var coords = skip_utrs ? 'pos_coding_noutr' : 'pos_coding';
    var max_cov = (metric == 'mean' || metric == 'median') ? 300 : 1; //sandeep
    var coding_coordinate_params = get_coding_coordinate_params(_transcript, skip_utrs);
    var chart_width;
    if (scale_type == 'overview') {
        chart_width = gene_chart_width;
    } else {
        chart_width = coding_coordinate_params.size*2;
    }

    var exon_x_scale = d3.scale.linear()
        .domain([0, coding_coordinate_params.size])
        .range([0, chart_width]);

    var svg = d3.select(container).select('#inner_svg')
        .attr("width", chart_width + gene_chart_margin.left + gene_chart_margin.right)
        .attr("height", gene_chart_height + gene_chart_margin.top + gene_chart_margin.bottom)
        .select('#inner_graph');

    var y = d3.scale.linear()
        .domain([0, max_cov])
        .range([gene_chart_height, 0]);

    var area = d3.svg.area()
        .x( function(d) {
            return exon_x_scale(d[coords]);
        }).y0( function(d) {
            return gene_chart_height;
        }).y1( function(d) {
            return (metric in d) ? y(d[metric]) : gene_chart_height;
        });


    var path = svg.selectAll("path")
        .datum(new_data)
        .transition()
        .duration(500)
        .attr("d", area)
        .style("fill", "steelblue")
        .attr('class', 'area');

    svg.selectAll("circle")
        .data(data)
        .transition()
        .duration(500)
        .attr("cx", function(d) {
            if (d[coords] == undefined) {
                return -100;
            } else {
                return exon_x_scale(d[coords]);
            }
        });

    // plot exons
    var svg_outer = d3.select(container).select('#track_svg')
        .attr("width", chart_width + gene_chart_margin_lower.left + gene_chart_margin_lower.right)
        .attr("height", lower_gene_chart_height).select('#track');

    svg_outer.select('#boundary_line')
        .attr("x2", exon_x_scale(coding_coordinate_params.size));

    // plot exon rounded rects
    svg_outer.selectAll("rect")
        .data(_transcript.exons)
        .transition()
        .duration(500)
        .attr("x", function(d, i) { return exon_x_scale(get_coding_coordinate(_transcript, d.start, skip_utrs)); })
        .attr("width", function(d, i) {
            if (get_coding_coordinate(_transcript, d.start, skip_utrs) == -100) {
                return exon_x_scale(175);
            }
            return exon_x_scale(d.stop-d.start+1);
        })
        .attr("height", function(d, i) {
            if (d.feature_type == 'CDS') {
                return lower_gene_chart_height;
            } else {
                return lower_gene_chart_height/2;
            }
        });

    // plot variants
    svg_outer.selectAll("a")
        .data(variant_data)
        .transition()
        .duration(500)
        .selectAll('ellipse')
        .attr("cx", function(d) {
            if (d[coords] == undefined) {
                return -100;
            } else {
                return exon_x_scale(d[coords]);
            }
        });

    var yAxis = d3.svg.axis()
        .scale(y)
        .orient("left");

    svg.select(".y.axis")
        .transition()
        .duration(200)
        .call(yAxis);
}

function create_new_data(data, coords) {
    var data_object = {};
    $.each(data, function(i, d) {
        data_object[d[coords]] = d;
    });
    var new_data = [];
    var max_coord = d3.max(data, function(d) { return d[coords] });
    for (var i = d3.min(data, function(d) { return d[coords] }); i < max_coord; i++) {
        var x = {'has_coverage': false};
        x[coords] = i;
        //Check the previous base to see if this is the beginning of an exon

        if (i in data_object && !(i-1 in data_object)) {
            new_data.push(x);
        }
        //Check the previous base to see if this is the end of an exon

        if (!(i in data_object) && i-1 in data_object) {
            x[coords] = i-1;
            new_data.push(x);
        }
        if (i in data_object) {
            new_data.push(data_object[i]);
        } else {
            new_data.push(x);
        }
    }
    return new_data;
}

function coverage_sum(key) {
    var total = 0;
    $.map(window.coverage_stats, function(entry) {
        total += entry[key];
    });
    return (total/window.coverage_stats.length).toPrecision(4);
}

function refresh_links() {
    $("#coverage_plot_download").attr('href', set_plot_image('gene_plot_container', 0));
    $("#exon_plot_download").attr('href', set_plot_image('gene_plot_container', 1));
}

$(document).ready(function() {
    if ($(window).width() < 768) {
        $('#gene_plot_container').css('width', $(window).width() + "px");
    } else {
        $('#gene_plot_container').css('width', $(window).width()*10/12 + "px");
    }
    precalc_coding_coordinates(window.transcript, window.coverage_stats, 'pos');
    precalc_coding_coordinates(window.transcript, window.variants_in_transcript, 'pos');

    // only show variants that have a coding coordinate
    window.variants_in_transcript = _.filter(window.variants_in_transcript, function(variant) {
        return variant.pos_coding != undefined;
    });

    // only show coding rects that have a coding coordinate
    window.coverage_stats = _.filter(window.coverage_stats, function(d) {
        return d.pos_coding != undefined;
    });
    $('#avg_coverage').html(coverage_sum('mean'));
    $('#avg_coverage_x').html(coverage_sum('30')*100 + '%');

    var new_data = create_new_data(window.coverage_stats, 'pos_coding');
    var new_data_skip_utr = create_new_data(window.coverage_stats, 'pos_coding_noutr');
    if (window.coverage_stats != null) {
        gene_chart(window.coverage_stats, new_data_skip_utr, window.variants_in_transcript, window.transcript);
        if (window.variants_in_transcript.length) {
            update_variants();
        }
        $('#loading_coverage').hide();
    } else {
        $('#gene_plot').hide();
        $('#not_covered').show();
    }
    // Change coverage plot
    $('.coverage_metric_buttons').change(function () {
        var v = $(this).attr('id').replace('_covmet_button', '');
        $('.coverage_subcat_selectors').hide();
        if (v == 'covered') {
            $('#over_x_select_container').show();
            v = $('#over_x_select').val().replace('X', '');
        } else {
            $('#average_select_container').show();
            v = $("#average_select").val();
        }
        var detail = $('.display_coverage_metric_buttons.active').attr('id').replace('display_coverage_', '').replace('_button', '');
        var include_utrs = $('#include_utrs_checkbox').is(':checked');
        var plot_data = include_utrs ? new_data : new_data_skip_utr;
        change_coverage_chart(window.coverage_stats, plot_data, window.variants_in_transcript, window.transcript, detail, v, !include_utrs, '#gene_plot_container');
        refresh_links();
    });
    $('#over_x_select').change(function () {
        var detail = $('.display_coverage_metric_buttons.active').attr('id').replace('display_coverage_', '').replace('_button', '');
        var include_utrs = $('#include_utrs_checkbox').is(':checked');
        var plot_data = include_utrs ? new_data : new_data_skip_utr;
        $('#avg_coverage_type_x').html($(this).val());
        $('#avg_coverage_x').html(coverage_sum($(this).val().replace('X', ''))*100 + '%');
        change_coverage_chart(window.coverage_stats, plot_data, window.variants_in_transcript, window.transcript, detail, $(this).val().replace('X', ''), !include_utrs, '#gene_plot_container');
        refresh_links();
    });
    $('#average_select').change(function () {
        var detail = $('.display_coverage_metric_buttons.active').attr('id').replace('display_coverage_', '').replace('_button', '');
        var include_utrs = $('#include_utrs_checkbox').is(':checked');
        var plot_data = include_utrs ? new_data : new_data_skip_utr;
        $('#avg_coverage_type').html($(this).val());
        $('#avg_coverage').html(coverage_sum($(this).val()));
        change_coverage_chart(window.coverage_stats, plot_data, window.variants_in_transcript, window.transcript, detail, $(this).val(), !include_utrs, '#gene_plot_container');
        refresh_links();
    });
    $('#include_utrs_checkbox').change(function () {
        setTimeout(function() {
            var detail = $('.display_coverage_metric_buttons.active').attr('id').replace('display_coverage_', '').replace('_button', '');
            var v = $('.coverage_metric_buttons.active').attr('id').replace('_covmet_button', '');
            v = (v == 'covered') ? $('#over_x_select').val().replace('X', '') : $("#average_select").val();
            var include_utrs = $('#include_utrs_checkbox').is(':checked');
            var plot_data = include_utrs ? new_data : new_data_skip_utr;
            change_coverage_chart(window.coverage_stats, plot_data, window.variants_in_transcript, window.transcript, detail, v, !include_utrs, '#gene_plot_container');
            refresh_links();
        }, 10);
    });

// Change exon diagram
    $('#inverted_checkbox').change(function () {
        setTimeout(function () {
            var v = $('#inverted_checkbox').is(':checked');
            change_track_chart_variant_size(window.variants_in_transcript, v, '#gene_plot_container');
            refresh_links();
        }, 10);
    });

    $('.consequence_display_buttons, #filtered_checkbox').change(function () {
        setTimeout(function() {
            update_variants();
            refresh_links();
        }, 10);
    });

    $('.display_coverage_metric_buttons').change(function () {
        var detail = $(this).attr('id').replace('display_coverage_', '').replace('_button', '');
        var v = $('.coverage_metric_buttons.active').attr('id').replace('_covmet_button', '');
        if (v == 'covered') {
            $('#over_x_select_container').show();
            v = $('#over_x_select').val().replace('X', '');
        } else {
            $('#average_select_container').show();
            v = $("#average_select").val();
        }
        var include_utrs = $('#include_utrs_checkbox').is(':checked');
        var plot_data = include_utrs ? new_data : new_data_skip_utr;
        change_coverage_chart(window.coverage_stats, plot_data, window.variants_in_transcript, window.transcript, detail, v, !include_utrs, '#gene_plot_container');
        refresh_links();
    });
    refresh_links();
});
</script>
                    
                </div>
                <div class="row">
                    
                        No variants in this gene.
                    
                </div>
            </div>
        </div>
    </div>

</body>














